test/ios_application_test.sh

```diff
@@ -351,12 +353,26 @@ EOF
 
     readonly FILE_TO_CHECK="${TEST_TMPDIR}/dumped_entitlements"
     readonly SHOULD_CONTAIN="${FOR_SIM}"
+
+    # Simulator builds also have entitlements in the codesign output,
+    # but only `com.apple.security.get-task-allow` and nothing else
+    do_build ios "$@" //app:dump_codesign || fail "Should build"
+
+    if [[ "${SHOULD_CONTAIN}" == "y" ]] ; then
+      assert_contains "<key>com.apple.security.get-task-allow</key>" "${CODESIGN_OUTPUT}"
+      assert_not_contains "<key>keychain-access-groups</key>" "${CODESIGN_OUTPUT}"
+    else
+      assert_not_contains "<key>com.apple.security.get-task-allow</key>" "${CODESIGN_OUTPUT}"
+      assert_not_contains "<key>keychain-access-groups</key>" "${CODESIGN_OUTPUT}"
+    fi
   fi
 
   if [[ "${SHOULD_CONTAIN}" == "y" ]] ; then
     assert_contains "<key>get-task-allow</key>" "${FILE_TO_CHECK}"
+    assert_contains "<key>keychain-access-groups</key>" "${FILE_TO_CHECK}"
   else
     assert_not_contains "<key>get-task-allow</key>" "${FILE_TO_CHECK}"
+    assert_contains "<key>keychain-access-groups</key>" "${FILE_TO_CHECK}"
```

Comment:
Added these to verify that the existing entitlements are included with the defaults.